### ISSUE TITLE
Catchem should NOT be free

### ISSUE THREAD

**Issue by thanatos86** [Lunat1q/Catchem-PoGo#2284]
I'm currently paying to use Pokefarmer. The money that they make probably helps them to crack API's.

Pokefarmer is pure garbage. No matter what I do, my trainer sticks in one tiny area, alwyas skipping pokemon, alwyas returning to the poketsop it just spun 60 seconds prior. It never ventures away form a tiny, tiny radiius even if the radius is unlimited. If you tell it to walk to a specific point, it does nothing but walk. It won't catch anything or loot any pokestops. Once it reaches it's destination, it walks right back to the point that you wanted it to leave. There's no way in hell that tard bot will ever be able ot create a custom route like this one. That bot is pure garbage compared to this one.

I have learned that Catchem is by far the greatest bot by a landslide.

No way in hell this bot should be free. No way in hell.

Maybe if we all forked over $5 per 1 month sesson (the cost of a damn hamburger!!!!!) you guys could afford to hire help to crack the API. I dunno.

I just don't think this bot should be free because of how amazing it is compared to the garbage that's out there.

**Comment by Furtif** [comment]
Not everything that is paying is necessarily better, necrobot, this bot ci, ninja, hackemon, rockect bot are free and some of them well designed.

**Comment by thanatos86** [comment]
I never implied that they were.

**Comment by Furtif** [comment]
@thanatos86 Exactly, the problem is that "paying" bots are a certainty that work, but in reality one finances an organization or members of the current application, giving them ideas etc. that will eventually be Blocked has all the others .. I know not .. but to make profit in hackent a protected application I do not see the interest to do so in "paying" mode.
